### ISSUE TITLE
R Session Aborted

### ISSUE THREAD

**Issue by vivamoto** [suiji/Arborist#51]
I was training a data set with the Rborist library when R crashed.

**Code:**
```{r}
library(dslabs)
library(Rborist)
library(caret)

mnist <- read_mnist()
index <- sample(nrow(mnist$train$images), 10000)

x <- mnist$train$images[index,]
y <- factor(mnist$train$labels[index])
colnames(x) <- 1:ncol(mnist$train$images)
col_index <- setdiff(1:ncol(x), nearZeroVar(x))

control <- trainControl(method="cv", number = 5, p = 0.8)
grid <- expand.grid(minNode = c(1,5) , predFixed = c(10, 15, 25, 35, 50))
train_rf <-  train(x[, col_index], y,
                   method = "Rborist",
                   nTree = 50,
                   trControl = control,
                   tuneGrid = grid,
                   nSamp = 5000)
```
**Message:**
> R Session Aborted
> R encountered a fatal error.
> The session was terminated.

**Additional info:**
RStudio Version 1.2.5001, Build 1468 (7b3fe265, 2019-09-18)
 ```{r}
> library(Rborist)
Rborist 0.2-2
Type RboristNews() to see new features/changes/bug fixes.

> version
               _                           
platform       x86_64-w64-mingw32          
arch           x86_64                      
os             mingw32                     
system         x86_64, mingw32             
status                                     
major          3                           
minor          6.1                         
year           2019                        
month          07                          
day            05                          
svn rev        76782                       
language       R                           
version.string R version 3.6.1 (2019-07-05)
nickname       Action of the Toes    
```

**Comment by suiji** [comment]
Unable to reproduce, so far.

Could you please try adding "autoCompress = 1.0" (unquoted) to the list of arguments passed to Rborist?  This is a workaround for a known bug, and may or may not solve your problem.  If so, the problem has been fixed in 0.2-3, which should be on CRAN in the next few weeks.

**Comment by suiji** [comment]
No response.
